common/djangoapps/philu_commands/management/commands/tests/test_archive_past_courses_communities_command.py

```diff
@@ -0,0 +1,98 @@
+"""
+Unit Tests for Django management command to archive course community on nodeBB.
+"""
+from __future__ import unicode_literals
+
+from datetime import datetime, timedelta
+
+import mock
+import pytz
+from django.core.management import call_command
+
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+
+
+class ArchivePastCoursesCommunities(ModuleStoreTestCase):
+    """
+    Tests for `archive_past_courses_communities` command.
+    """
+
+    def setUp(self):
+        """
+        This function is responsible for creating courses for every test and mocking the function for tests.
+        :return:
+        """
+        super(ArchivePastCoursesCommunities, self).setUp()
+
+        self.course_1 = CourseFactory.create(display_name='test course 1', run='Testing_course_1')
+        self.course_2 = CourseFactory.create(display_name='test course 2', run='Testing_course_2')
+        self.course_1.end = datetime.now(pytz.UTC) - timedelta(hours=2)
+        self.course_2.end = datetime.now(pytz.UTC) - timedelta(hours=2)
+
+        self.get_visible_courses_patcher = \
+            mock.patch('philu_commands.management.commands.archive_past_courses_communities.get_visible_courses')
```

Comment:
Can we avoid \ by introducing a local variable

```
get_visible_courses = 'philu_commands.management.commands.archive_past_courses_communities.get_visible_courses'

self.get_visible_courses_patcher = mock.patch(get_visible_courses)
```

Reply:
modified